src/index.html

```diff
@@ -10,13 +10,114 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="./reset.css"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        />
+      </a>
+
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a
+              href="/apple"
+              class="is-active nav__item"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__link">
+            <a
+              href="/samsung"
+              class="nav__item"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__link">
+            <a
+              href="/smartphones"
+              class="nav__item"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__link">
+            <a
+              href="/laptop-Computers"
```

Comment:
The URL in the href attribute should be in lowercase to follow standard URL conventions. Consider changing '/laptop-Computers' to '/laptop-computers'.